Src/Base/AMReX_Extension.H

```diff
@@ -190,6 +190,20 @@
 #    define AMREX_FALLTHROUGH ((void)0)
 #endif
 
+// Note: following compilers support [[likely]] and [[unlikely]]
+//   - Clang >= 12.0
+//   - GCC >= 9.0
+//   - Intel >= 2021.7
+//   - MSVC >= 19.26
+//   - nvcc >= 12
+#if defined(__has_cpp_attribute) && __has_cpp_attribute(likely) >= 201803L
+#    define AMREX_LIKELY [[likely]]
+#    define AMREX_UNLIKELY [[unlikely]]
+#else
+#    define AMREX_LIKELY ((void)0)
```

Comment:
Should this be empty instead of `((void)0)`?

Reply:
yes.  good point